src/lib/RbtDihedralSF.cxx

```diff
@@ -11,6 +11,7 @@
  ***********************************************************************/
 
 #include "RbtDihedralSF.h"
+#include <cmath>
 
```

Comment:
```suggestion
#include <cmath>

#include "RbtDihedralSF.h"

```

Reply:
Feel free to directly push any necessary changes to this branch.
("Allow edits by maintainers" is checked.)